lib/Providers/Time/HttpTimeProvider.php

```diff
@@ -21,10 +21,8 @@ class HttpTimeProvider implements ITimeProvider
     public function __construct(
         public string $url = 'https://google.com',
         public string $expectedtimeformat = 'D, d M Y H:i:s O+',
-        array $options = null,
+        ?array        $options = null,
```

Comment:
missing explicit public/private and potentially annoying spaces as previously mentioned

Reply:
`$options` is now defined by constructor property promotion in https://github.com/RobThree/TwoFactorAuth/pull/108/commits/c2183e16b372bf7c3303d437f00a326ca964771e.
As for the formatting, that's PHP-CS-Fixer's doing.